src/Meta/Collection.php

```diff
@@ -48,14 +48,18 @@ public static function get_collection_title( $post_type ) {
 		$post_type_name = 'Blog';
 
 		if ( 'post' !== $post_type ) {
-			$post_type = get_post_type_object( $post_type );
-
-			$post_type_name = $post_type->labels->name;
+			$type = get_post_type_object( $post_type );
+			$post_type_name = $type->labels->name;
 		}
 
-		return apply_filters(
-			sprintf( self::TITLE_FILTER, $post_type ),
-			$post_type_name . ' - ' . get_bloginfo( 'title' )
+		$filter_name = sprintf(
+			self::TITLE_FILTER,
+			is_string( $post_type ) ? $post_type : $post_type_name
+		);
```

Comment:
Yes, it does.

How about my 2nd comment?

is_string() will always return true